cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/DefaultNamespaceAdmin.java

```diff
@@ -130,13 +130,17 @@ public synchronized void create(NamespaceMeta metadata) throws Exception {
       authorizerInstantiator.get().enforce(instanceId, principal, Action.ADMIN);
     }
 
+    // store the metadata first and then create namespaces in the storage handler.
+    // TODO (CDAP-6155): this will be switched back to original when we do hbase mapping where we will start passing the
+    // NamespaceMeta itself to the DatasetFramework. We should first create namespaces in underlying storage before
```

Comment:
The config does not have the namespace if so I dont think its a good idea to pass  just the config.